app/models/concerns/rubygem_searchable.rb

```diff
@@ -37,9 +40,13 @@ def as_indexed_json(_options = {})
              }
 
     mapping do
-      indexes :name, analyzer: 'rubygem'
-      indexes :indexed, type: 'boolean'
-      indexes :info, analyzer: 'english'
+      indexes :name, type: 'multi_field' do
```

Comment:
For some reason it didn't seem to work without `multi_field` but it's possible I did something else wrong too.